Refresh Rancher Desktop install of Epinio (#307)
epinio/docs: docs/installation/other_inst_scenarios/install_epinio_on_rancher_desktop.md

---
sidebar_label: "Installing Epinio on Rancher Desktop"
sidebar_position: 1
title: "Installing Epinio on Rancher Desktop"

> description: iHow o install Epinio using a local Rancher Desktop installation.

Reviewer: ```suggestion
description: How to install Epinio using a local Rancher Desktop installation.
```

The Epinio installation is similar on Linux, MacOS and Windows:

> <!--TODO: Any particular reason why the Windows sheel prefers the latest?-->

Reviewer: Looking through the history this part was written by @ldevulder .

Author: Thanks.

For RancherDesktop on Linux, in order to be able to open ports from `443` (and above), in order to access the URL set in `global.domain` (i.e. 127.0.0.1.sslip.io), you need to set the start port of the `unprivileged` list to a lower number:

<!--

> TODO:Does this need to be 80 as per the RD installation docs?

Reviewer: Looking through the history this part came from @nunix .

Author: Thanks.
